File: condicionales.py
# Condicionales

""" 

if condicion:
    sentencias
    
if condicion:
    sentencias
else:
    sentencias

if condicion:
    sentencias
elif condicion:
    sentencias
else:
    sentencias
    
and 
or
not

"""

a = 5
b = 10
c = 4


if a > b:
    print("Info")
    
if b == c:
    pass
else:
    pass

if a > b and a > c:
    print("A")
elif b > c:
    print("B")
else:
    print("C")
    
frutas = { "Pera", "Manzana", "Banana", "Uva"}
nombres = ["Hugo", "Paco", "Luis"]

if "Manzana" in frutas:
    print("Existe")
    
if "Hugo" in nombres:
    print("Hugo esta presente")
    
# ternario
mayor = "A" if a > b else "B"

File: condicionales.js
// Condicionales
/* 

if (condicion) {
    sencencias
}

if (condicion) {
    sentencias
} else {
    sentencias
}

if (condicion) {
    sentencias
} else if (condicion) {
    sentencias
} else {
    sentencias
}

switch(value){
    case
    default
}

and = &&
or = ||
not = !

*/

let a = 5;
let b = 10;
let c = 4;

if (a > b){
    console.log("Info")
}


if ( b === c) {
    // sentencias
} else {
    // sentencias
}

if (a > b && a > c) {
    console.log("A")
} else if (b > c){
    console.log("B")
} else {
    console.log("C")
}

// ternario
let mayor = a > b ? "A" : "B"